.env.example

```
@@ -26,3 +26,5 @@ IS_AI_ENABLED='true'
 IS_NOTIFICATION_SERVICE_ENABLED='true'
 
 GOOGLE_API_KEY=''
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)